apps/calculation/app.cpp

```diff
@@ -10,11 +10,11 @@ using namespace Shared;
 namespace Calculation {
 
 I18n::Message App::Descriptor::name() {
-  return I18n::Message::CalculApp;
+  return I18n::Message::CalculationApp;
```

Comment:
Why do you need to rename a fuck tone of i18n string? This should be part of a different commit, or even a different PR. Furthermore, this can cause future issues with newer versions of epsilon when trying to merge things. I'd consider reverting the rename changes.

Reply:
Because, the program need apps name in a const array list, so a generate this const list via a macro but the program generate the name with this pattern: "<Name_of_the_app>App", so some apps like calculation don't follow this pattern so I change that

Reply:
Yeah but as maxime says it generate a issue while merging latest version of epsilon, and it take long to fix it :-/

Reply:
If I keep CalculApp and add CalulationApp, does the program interpret that as an addition instead of an modification that cause problems and does that fix the problem ? (We could even keep CalculApp in apps descriptor to decrease risk of conflict)